hack/functests.sh

```diff
@@ -45,6 +45,11 @@ function functest() {
         echo "Will skip test asserting the cluster is in dual-stack mode."
         extra_args="${extra_args} -skip-dual-stack-test"
     fi
+
+    extra_args="${extra_args} -conn-check-ipv4-address=${conn_check_ipv4_address:+-$conn_check_ipv4_address}"
```

Comment:
`conn-check-ipv4-address=` the var itself should not be part of the line in case the value is `""`
also the '-' is wrong there, instead the `-` please put `conn-check-ipv4-address=` string i think

Reply:
>  so the '-' is wrong there, instead the - please put conn-check-ipv4-address= string i think

please look [here] (https://github.com/kubevirt/kubevirt/blob/e7d3b15e900d9ee0e57d63c250374ade2237d4ab/hack/functests.sh#L53)
all the othre flags do use `-`. 

> conn-check-ipv4-address= the var itself should not be part of the line in case the value is ""

Unfortunately my bash skills are lame. Can you right the exact line you think I should use?

Reply:
Related to
https://github.com/kubevirt/kubevirt/pull/5169#discussion_r625040486
In case the direction you choose at this comment is to change the defaults you must not send the values if they are empty

example 
```
$ VAR=8.8.8.8
$ echo ${VAR:+-var=$VAR}
-var=8.8.8.8
$ VAR=""
$ echo ${VAR:+-var=$VAR}

$ 
```

Reply:
I prefer not to use this direction. I wrote in the comment there why. 
As for the script . I'll see if there is something better I can use.

Reply:
Np
I mean 
`extra_args="${extra_args} -conn-check-ipv4-address=${conn_check_ipv4_address:+-$conn_check_ipv4_address}"`
this doesnt make sense, because if the var is nil it will be nil anyhow so either do what you did in the very beginning
`extra_args="${extra_args} -conn-check-ipv4-address=${conn_check_ipv4_address"`
or do as in the example in the command above, where the `-conn-check-ipv4-address=` itself will be printed only if the var is not empty please

Reply:
I returned to the version in that was at the beginning. I don't think it is that bad since we check the flag value in the test code.